vms/rpcchainvm/vm_server.go

```diff
@@ -363,6 +363,30 @@ func (vm *VMServer) CreateHandlers(ctx context.Context, _ *emptypb.Empty) (*vmpb
 	return resp, nil
 }
 
+func (vm *VMServer) CreateGRPCService(ctx context.Context, _ *emptypb.Empty) (*vmpb.CreateGRPCHandlerResponse, error) {
+	serviceName, handler, err := vm.vm.CreateGRPCService(ctx)
```

Comment:
That looks like a bug. I think it's not caught in CI because the only vm that uses rpcchainvm in our testing as part of this PR does register a grpc service.